### ISSUE TITLE
SimpleEnchantment non-alphabet enchantments 

### ISSUE THREAD

**Issue by randomUser134923** [kangarko/Foundation#111]
Currently, SimpleEnchantment is registering Enchantment with 
```
    private static NamespacedKey toKey(@NonNull String name) {
        if (name == null) {
            throw new NullPointerException("name is marked non-null but is null");
        } else if (!MinecraftVersion.atLeast(V.v1_13)) {
            throw new RuntimeException("SimpleEnchantment requires Minecraft 1.13.2 or greater. Cannot make " + name);
        } else {
            name = new String(name);
            name = name.toLowerCase().replace(" ", "_");
            name = ChatUtil.replaceDiacritic(name);
            Valid.checkBoolean(name != null && VALID_NAMESPACE.matcher(name).matches(), "Enchant name must only contain English alphabet names: " + name, new Object[0]);
            return new NamespacedKey(SimplePlugin.getInstance(), name);
        }
    }
```
this checks if name is in alphabetic form because it's using name as namespacekey.

Instead doing something like
```
            UUID uuid = UUID.nameUUIDFromBytes(name.getBytes());
            return new NamespacedKey(SimplePlugin.getInstance(), uuid.toString());
```
 may be able to store unique key for enchantment while accept non-alphabet Enchantment names

**Comment by kangarko** [comment]
Have you tested this? I believe this is required by Spigot

**Comment by randomUser134923** [comment]
> Have you tested this? I believe this is required by Spigot

Actually yes, but kinda no. I confirmed that it does register into Enchantment by checking ``Enchantment.values()`` but it seems to not appear on client side when I apply it. I just tried by copy pasting SimpleEnchantment into new class of my plugin and changed getNamespacekey part. However, it's not the matter of above but there must be something going on behind that I can't check - as simply copy pasting everything from SimpleEnchantment and trying with that new class still doesn't work while using SimpleEnchantment works. 

If you can let me know how does registering Enchantment + make it visible to client briefly, I can try confirm if it works. 

Bottom line two line summary
1. Enchantment does get registered. checked by Enchantment.values(); 
2. However, I don't know how to properly make registered Enchantment work in client side too so couldnt' check if it actually work.

**Comment by kangarko** [comment]
Client won't register custom enhancements because it does not have the rendering mechanism for those. We can only fake them using our SimpleEnchantment wrapper and a packet listener we have in Foundation.

You cannot use non standard letters in enchantmnets keys, I found this is a rule by Spigot and the native code.
